test/inductor/test_flex_attention.py

```diff
@@ -525,6 +525,30 @@ def run_automatic_dynamic_test(
     def test_builtin_score_mods(self, dtype: torch.dtype, score_mod: Callable):
         self.run_test(score_mod, dtype)
 
+    @supported_platform
+    @patch.object(torch._inductor.config, "force_use_flex_attention", True)
```

Comment:
I think I would prefer for this to be a kernel option passed to `flex_attention`.

Reply:
Make sense, will update it.